test/integration/maincluster_test.go

```diff
@@ -893,6 +898,8 @@ experimental:
     enabled: true
   clusterAutoscalerSupport:
     enabled: true
+  clusterTLSBootstrap:
```

Comment:
Sry but the last nit; Would you mind informing me if there's any specific reason you've prefixed the key with "cluster"? Can we call it just `tlsBootstrap` for consistency with the k8s doc?